src/common/serviceInstanceHelpers.js

```diff
@@ -148,6 +150,20 @@ const handleWalkthroughOneRoutes = (namespacePrefix, dispatch, event) => {
   }
 };
 
+const handleServiceInstancesProvision = (namespacePrefix, dispatch, event) => {
+  const si = event.payload;
+  const lastCondition = _.last(si.status.conditions);
```

Comment:
Prefer if we didn't use lodash and just got the last element using plain javascript.